jobs/settings/__init__.py

```diff
@@ -2,6 +2,7 @@
 from datetime import timedelta
 
 import environ
+from collections import OrderedDict
```

Comment:
Hi @wolframtheta this line can be removed because `OrderedDict` is not used.

Reply:
Yep true, change done! I will try to check always the imports to avoid unused imports!